src/VisualizationTests/WpfVisualizationTests.csproj

```diff
@@ -1,4 +1,4 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
how are we running  tests for a wpf project ?
Or what is this change about ?
I see there is also a new Setup.cs file ? why ?

Reply:
because now `Moq` (v 4.18) has new dependencies which I only copy to the `test/test_dependencies` folder. So we need to setup a setupfixture resolve handler to find them.

Reply:
It just seemed to me that most of the Visualization tests are derived from SystemTestBase which already uses an AssemblyResolver which we could use

Reply:
yes, but not all of them - then we'd have two places to remember to look for this resolve to occur.